<AddItem/> - add subtitle prop

egidijusja opened the pull request:
<!---
Thanks for submitting a pull request 😄 !
-->

<!---
- Be as descriptive as possible when explaining what was changed.
- Link to an issue if one exists
-->

### 🔦 Summary

...

<!--- Please mark all checkbox. If one is not relevant - delete it -->

### ✅ Checklist

- [ ] 👨‍💻 API change is approved by the UI Infra Developers <!--- Please tag the relevant team member -->
- [ ] 👨‍🎨 UX change is approved by the Design System UX <!--- Please tag the relevant team member -->
- 📚 Change is documented
  - [ ] Story
  - [ ] API description
  - [ ] Cheatsheet
  - [ ] Other (explain)
- 🔬 Change is tested
  - [ ] Component
  - [ ] Visual test

WixBuildServer commented:
### 🤖 Hi, I'm #Janet!

  A deploy preview of your PR was successfully released 🖼.

  You can find it under one of the following URLs:
  * https://preview.wix-style-react.com/pr_6699

egidijusja commented:
Hey @milkyfruit, we have added subtitle prop for `addItem` component. In this PR I have added new subtitle example, could you review and give better description for it?

milkyfruit reviewed (changes requested):
@egidijusja two changes are needed:
- [x] Subtitle color should be `D20`
- [x] Please change example to:
### Subtitle
Component has a `subtitle` prop, it's meant to explain how "add" action works. This is especially useful when AddItem is used together with [Dropzone](url) component and it's important to list supported file formats.
```
<Layout gap={10} cols={1}>
  <AddItem
    size="large"
    subtitle="You can upload jpeg, png and gif files up to 5 mb size"
  >
    Drag here to upload
  </AddItem>
</Layout>;
```

mykas commented on packages/wix-style-react/src/AddItem/AddItem.st.css:
```diff
@@ -245,6 +254,24 @@
   color: value(D10-30);
 }
 
+.subtitle {
```
I see the purpose of this classname is to control the padding between main content and subtitle. Because you already created a class called `contentBody` - I am wondering why not to have the padding controlled there? You wouldn't need this subtitle classname anymore.

mykas commented on packages/wix-style-react/src/AddItem/AddItem.st.css:
```diff
@@ -245,6 +254,24 @@
+  padding-top: 6px;
+}
+
+.subtitle:skin(standard) {
```
oh I see there is another reason to have this.

mykas reviewed (approved):
Looks good!
